src/main/java/ua/tc/marketplace/exception/model/MappingException.java

```diff
@@ -0,0 +1,11 @@
+package ua.tc.marketplace.exception.model;
+
+public class MappingException extends RuntimeException {
```

Comment:
it should extend CustomRuntimeException. See `DemoNotFoundException` for example